### PR TITLE
chore: article design review

### PR DESCRIPTION
**Pull request by jimhunty**
Design review on Articles - Mobile only.

### CODE REVIEW

File: packages/article-main-standard/src/styles/article-meta/shared.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const sharedStyles = {
     marginRight: 0
   },
   articleMiddleContainer: {
-    paddingTop: 9
+    // paddingTop: spacing(2.5)
```
Comment (aronshamash) [inline packages/article-main-standard/src/styles/article-meta/shared.js]: remove or uncomment

File: packages/article-main-standard/src/styles/article-header/index.js [hunk 2]
```diff
@@ -8,19 +8,18 @@ const nativeStyles = {
...
   },
-  flags: {
-    ...sharedStyles.flags,
-    marginTop: spacing(1)
-  },
   standFirst: {
     ...sharedStyles.standFirst,
     lineHeight: 25
+  },
+  standFirstWithoutFlags: {
```
Comment (aronshamash) [inline packages/article-main-standard/src/styles/article-header/index.js]: sure `standfirst` is one word.

File: packages/article-main-comment/__tests__/ios/__snapshots__/article-with-style.ios.test.js.snap [hunk 3]
```diff
@@ -48,8 +48,8 @@ exports[`phone full article with style 1`] = `
             "color": "#1D1D1B",
             "fontFamily": "TimesModern-Bold",
             "fontSize": 30,
-            "lineHeight": 35,
-            "marginBottom": 10,
+            "lineHeight": 33,
+            "marginBottom": 12.5,
```
Comment (aronshamash) [inline packages/article-main-comment/__tests__/ios/__snapshots__/article-with-style.ios.test.js.snap]: is this desired ?

File: packages/article-main-comment/__tests__/android/__snapshots__/article-with-style.android.test.js.snap [hunk 4]
```diff
@@ -174,8 +174,8 @@ exports[`tablet full article with style 1`] = `
             "color": "#1D1D1B",
             "fontFamily": "TimesModern-Bold",
             "fontSize": 30,
-            "lineHeight": 35,
-            "marginBottom": 10,
+            "lineHeight": 33,
+            "marginBottom": 13.75,
```
Comment (aronshamash) [inline packages/article-main-comment/__tests__/android/__snapshots__/article-with-style.android.test.js.snap]: don't we want whole numbers?

File: packages/article-main-standard/src/styles/article-header/index.js (hunk 2 above)
Comment (jimhunty) [reply packages/article-main-standard/src/styles/article-header/index.js]: Look at the property above. It's consistent.

File: packages/article-main-comment/__tests__/ios/__snapshots__/article-with-style.ios.test.js.snap (hunk 3 above)
Comment (jimhunty) [reply packages/article-main-comment/__tests__/ios/__snapshots__/article-with-style.ios.test.js.snap]: This is due to poor line heights within the font itself. Means margins around them need to be micromanaged to give the same illusion of space. It's not desirable but has to be there.

File: packages/article-main-comment/__tests__/android/__snapshots__/article-with-style.android.test.js.snap (hunk 4 above)
Comment (jimhunty) [reply packages/article-main-comment/__tests__/android/__snapshots__/article-with-style.android.test.js.snap]: See comment above. Basically poor fonts.

File: packages/article-main-standard/src/styles/article-meta/shared.js (hunk 1 above)
Comment (jimhunty) [reply packages/article-main-standard/src/styles/article-meta/shared.js]: Sorted.

File: packages/article-main-standard/src/styles/article-header/index.js (hunk 2 above)
Comment (jimhunty) [reply packages/article-main-standard/src/styles/article-header/index.js]: Does this constitute blocking this PR? Sins of the past...

**Pull request merged**